src/main/java/org/jabref/gui/fieldeditors/identifier/IdentifierEditor.java

```diff
@@ -82,9 +82,9 @@ public IdentifierEditor(Field field,
                 new Tooltip(Localization.lang("Look up %0", field.getDisplayName())));
 
         if (field.equals(DOI)) {
-            textArea.initContextMenu(EditorMenus.getDOIMenu(textArea, dialogService, preferencesService));
+            textArea.initContextMenu(EditorMenus.getDOIMenu(textArea, dialogService), preferencesService.getKeyBindingRepository());
```

Comment:
Shoulnd't the Injector be used to create a `KeyBindingRepository`

Reply:
see above